tests/regression_tests/regression.py

```diff
@@ -309,8 +309,8 @@ def run_derived_eigenfunction_test(self, derived_eigenfunction, ds_test, ds_base
             ds_base,
             names_attr="derived_ef_names",
             figname_prefix="derived_",
-            nb_plots=(5, 4),
-            figsize=(18, 10),
+            nb_plots=(5, 5),
+            figsize=(23, 10),
```

Comment:
I assume this was increased because there are now 21 derived eigenfunctions instead of 20?

Reply:
Indeed. I don't know if setting it to (3, 7) to have exactly 21 plots would be better.